octo/statik/statik.go

```diff
@@ -8,7 +8,7 @@ import (
0\x00$\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81\x8d\x14\x00\x00install-update-package-dependency.shUT\x05\x00\x01\x9an\xaddPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x8fk\xb9VZ}\x1a\x0e\xe4\x00\x00\x00E\x01\x00\x00\x0d\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81\x81\x15\x00\x00install-yj.shUT\x05\x00\x01~bodPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x08\x97\xeaV\x8f\xf8\x83n\xf2\x01\x00\x00,\x05\x00\x00\x14\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81\xa9\x16\x00\x00package-buildpack.shUT\x05\x00\x01PT\xacdPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x8fk\xb9V\xcd\x81\x9d\xa8\x08\x01\x00\x00M\x02\x00\x00\x12\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81\xe6\x18\x00\x00republish-image.shUT\x05\x00\x01~bodPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x8fk\xb9Vt4b\x1aN\x00\x00\x00W\x00\x00\x00\x18\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x817\x1a\x00\x00run-integration-tests.shUT\x05\x00\x01~bodPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x8fk\xb9V\xa4\xd5!O?\x00\x00\x00:\x00\x00\x00\x0c\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81\xd4\x1a\x00\x00run-tests.shUT\x05\x00\x01~bodPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x8fk\xb9V!\xdd\xeceF\x00\x00\x00D\x00\x00\x00\x11\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81V\x1b\x00\x00run-unit-tests.shUT\x05\x00\x01~bodPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x8fk\xb9V\xd9\xf2\x10&\x9e\x01\x00\x00b\x03\x00\x00 \x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81\xe4\x1b\x00\x00update-build-image-dependency.shUT\x05\x00\x01~bodPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x08\x97\xeaV\xc7\xb2\xfa\x19\x01\x02\x00\x00\xa6\x04\x00\x00 \x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81\xd9\x1d\x00\x00update-buildmodule-dependency.shUT\x05\x00\x01PT\xacdPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x08\x97\xeaV\xfeU\xc7\x96\xad\x01\x00\x00\xcc\x03\x00\x00\x0c\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x811 \x00\x00update-go.shUT\x05\x00\x01PT\xacdPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x8fk\xb9V\x05N6\xabh\x01\x00\x00\xf9\x02\x00\x00\x1e\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81!\"\x00\x00update-lifecycle-dependency.shUT\x05\x00\x01~bodPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x8fk\xb9V\xab\xefq\n{\x00\x00\x00z\x00\x00\x00\x0f\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xa4\x81\xde#\x00\x00update-netrc.shUT\x05\x00\x01~bodPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x8fk\xb9V{L\xed\xea\xeb\x01\x00\x00z\x05\x00\x00\x1c\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81\x9f$\x00\x00update-package-dependency.shUT\x05\x00\x01~bodPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\x8fk\xb9V\xa6^\xd1\xe8\x8b\x02\x00\x00\xad\x04\x00\x00\x12\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81\xdd&\x00\x00update-pipeline.shUT\x05\x00\x01~bodPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\xd7\xa4\xdeV\xbc\xfb5Z\x1e\x01\x00\x00{\x02\x00\x00\x18\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xed\x81\xb1)\x00\x00update-release-digest.shUT\x05\x00\x01W=\x9fdPK\x05\x06\x00\x00\x00\x00\x1f\x00\x1f\x00w	\x00\x00\x1e+\x00\x00\x00\x00"
```

Comment:
😢

Reply:
Aye.. statik.go contains the *.sh scripts from the project, so they can be output by octo later.. it's umm.. entertaining..